Sillycore.Web/Constants.cs

```diff
@@ -9,5 +9,7 @@ public class Constants
         public const string CertificatePassword = "CertificatePassword";
         public const string IsShuttingDown = "IsShuttingDown";
         public const string UseSwagger = "UseSwagger";
+        public const string IsSecure = "IsSecure";
```

Comment:
IsSecure bana biraz https mi acaba algısı uyandırıyor? Bunun adını daha uygun ne yapabiliriz?

Reply:
Esasında bizdeki kullanıma göre düşününce authentication ve authorization beraber ele alınacağı için böyle generic bir isim koydum. İkisini ayırıp daha spesifik olarak isimlerini authentication ve authorization verebiliriz. 
Böyle yek bir isim kalsın dersek de önerilerim: IsGuarded, IsProtected  ya da IsAccessControlled gibi şeyler olabiliriz.